src/app/task/done-tasks/done-tasks.component.scss

```diff
@@ -0,0 +1,13 @@
+h2 {
```

Comment:
Also try not to add styles based on html tag. instead create specific class and then add it to the element. This way if you ever add new h2 elements to the view they won't automatically get those styles. :)